GOVERNANCE.md

```diff
@@ -40,6 +40,7 @@ Have you done something (big or small) to contribute to the health, success, or
 #### Privileges
 
 - New role on [Discord](https://astro.build/chat): `@contributor`
+- Invitation to the private #contributors channel on Discord.
```

Comment:
Only nit is that I think we just closed this channel, is that correct @jonathantneal?

Reply:
```suggestion
```

Reply:
thanks, I was thinking about this but forgot to look into